src/pathPlanning/GCodePath.cpp

```diff
@@ -6,11 +6,12 @@
 namespace cura
 {
 
-GCodePath::GCodePath(const GCodePathConfig* config, SpaceFillType space_fill_type, float flow, bool spiralize) :
+GCodePath::GCodePath(const GCodePathConfig* config, SpaceFillType space_fill_type, float flow, bool spiralize, double speed_factor) :
 config(config),
 space_fill_type(space_fill_type),
 flow(flow),
-spiralize(spiralize)
+spiralize(spiralize),
+speed_factor(speed_factor)
```

Comment:
I'm getting a compile warning here:
"spiralize will be initialized after"

You should initialize the member variables in the same order as they are defined.